app/src/main/res/values/dimens.xml

```diff
@@ -8,6 +8,7 @@
     <dimen name="et_pass_margin_horizontal">16dp</dimen>
     <dimen name="et_mail_bottom">39dp</dimen>
     <dimen name="et_pass_bottom">216dp</dimen>
+    <dimen name="et_size">100sp</dimen>
```

Comment:
use dp here instead of sp.. you are using this dimen for a android:layout_width
You have you use sp only for texts

Reply:
done